pkg/kubectl/cmd/attach/attach.go

```diff
@@ -61,8 +61,8 @@ const (
 	defaultPodLogsTimeout   = 20 * time.Second
 )
 
-// AttachOptions declare the arguments accepted by the Exec command
-type AttachOptions struct {
+// Options declare the arguments accepted by the Attach command
+type Options struct {
```

Comment:
don't change these.

Reply:
@prksu Thanks for your feedback.
This is the default recommendation of `golint`.
Is there a better way to solve this problem?
I need more helpful advice.

Reply:
i once asked this before in the slack sig-cli channel. and it has been rejected before.
my advice just leave the current naming. see https://github.com/kubernetes/kubernetes/pull/70682#discussion_r233642556

Reply:
Thanks for your explanation. That make sense!

Reply:
Code updated. @prksu